modules/bg_controller/iam.tf

```diff
@@ -0,0 +1,80 @@
+# ---- iam role for Lambdas
```

Comment:
Same comment as for file appmesh-sf-task-token.py

Reply:
if it appears in master how come it appears as a change ?